web_src/js/modules/fomantic/dimmer.js

```diff
@@ -0,0 +1,29 @@
+import $ from 'jquery';
+import {queryElemChildren} from '../../utils/dom.js';
+
+export function initFomanticDimmer() {
+  // stand-in for removed dimmer module
+  $.fn.dimmer = function (arg0, $el) {
+    if (arg0 === 'add content') {
+      const existingDimmer = document.querySelector('body > .ui.dimmer');
+      if (existingDimmer) {
+        queryElemChildren(existingDimmer, '*', (el) => el.remove());
```

Comment:
This change doesn't seem right.

Reply:
Well we definitely need to remove any previous modals otherwise two would show with the `append` below.